<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVG Build</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="optionsGroupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="buildGroupBox">
         <property name="title">
          <string>Build</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="simplePathCheckBox">
            <property name="text">
             <string>Simple Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="fullPathCheckBox">
            <property name="text">
             <string>Full Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="textCheckBox">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="imageCheckBox">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="rangeGroupBox">
         <property name="title">
          <string>Range</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="fromLabel">
            <property name="text">
             <string>From</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="fromSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="untilLabel">
            <property name="text">
             <string>Until</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="untilSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>99999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="behaviourGroupBox">
         <property name="title">
          <string>Behaviour</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="pageCheckBox">
            <property name="text">
             <string>Page</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cameraCheckBox">
            <property name="text">
             <string>Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="backwardCheckBox">
            <property name="text">
             <string>Backward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="topCheckBox">
            <property name="text">
             <string>Top</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="colorGroupBox">
         <property name="title">
          <string>Color</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lineLabel">
            <property name="text">
             <string>Line</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="lineColorWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: black </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="lineToolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="cameraFrameColorWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: red</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="backgroundLabel">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="cameraFrameLabel">
            <property name="text">
             <string>Camera Frame</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="backgroundColorWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: white</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="backgroundToolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="cameraFrameToolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="markerLabel">
            <property name="text">
             <string>Marker</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="markerComboBox">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Diamond</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/diamond.png</normaloff>:/diamond.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Scissor</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/scissor.png</normaloff>:/scissor.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Triangle</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/triangle.png</normaloff>:/triangle.png</iconset>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="canvasGroupBox">
         <property name="title">
          <string>Canvas</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="heightLabel">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="heightSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1920</number>
            </property>
            <property name="value">
             <number>480</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="widthLabel">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="widthSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2560</number>
            </property>
            <property name="value">
             <number>640</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="othersGroupBox">
         <property name="title">
          <string>Others</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="zoomLabel">
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="zoomSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="folderNameLabel">
            <property name="text">
             <string>Folder Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="folderNameLineEdit">
            <property name="text">
             <string>movie</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="chooseFileGroupBox">
      <property name="title">
       <string>Choose File</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="openFileLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openFilePushButton">
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buildPushButton">
      <property name="text">
       <string>Build!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionSVGBuild_Help"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSetting">
   <property name="text">
    <string>&amp;Setting</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSVGBuild_Help">
   <property name="text">
    <string>SVGBuild &amp;Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>appendLine(QString)</slot>
  <slot>selectFile()</slot>
  <slot>openLineColorPicker()</slot>
  <slot>openBackgroundColorPicker()</slot>
  <slot>openFrameColorPicker()</slot>
  <slot>openSettingsDialog()</slot>
  <slot>proceedBuildUp()</slot>
  <slot>finished()</slot>
  <slot>resetThread()</slot>
 </slots>
</ui>
